playground/css-codesplit/__tests__/css-codesplit-consistent.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { beforeEach, describe, expect, test } from 'vitest'
+import { findAssetFile, isBuild, startDefaultServe } from '~utils'
+
+beforeEach(async () => {
+  await startDefaultServe()
+})
+
+for (let i = 0; i < 5; i++) {
+  describe.runIf(isBuild)('css-codesplit build', () => {
+    test('should be consistent with same content', () => {
+      expect(findAssetFile(/style-.+\.css/)).toMatch('h2{color:#00f}')
+      expect(findAssetFile(/style2-.+\.css/)).toBe('')
+    })
+  })
+}
```

Comment:
```suggestion
describe.runIf(isBuild)('css-codesplit build', () => {
  test('should be consistent with same content', () => {
    expect(findAssetFile(/style-.+\.css/)).toMatch('h2{color:#00f}')
    expect(findAssetFile(/style2-.+\.css/)).toBe('')
  })
})
```

This only runs the build once and then checks the same assets for five times.

Maybe we can put this test case in `css-codesplit.spec.ts` as it's one test case and doesn't require a separate vite config.

Reply:
Oh, my bad. You're right. I mixed up `beforeEach` and `beforeAll` in my head😅